### ISSUE TITLE
Bug in multiply_errorfree

### ISSUE THREAD

**Issue by dpsanders** [JeffreySarnoff/ErrorfreeArithmetic.jl#4]
```jl
julia> multiply_errorfree(Inf, Inf)
(Inf, NaN)
```
The second component should be 0, I guess.

**Comment by dpsanders** [comment]
There is the same issue with 
```
julia> multiply_errorfree(-Inf, -Inf)
(Inf, NaN)

julia> multiply_errorfree(-Inf, Inf)
(-Inf, NaN)
```

**Comment by dpsanders** [comment]
Fix in #5

**Comment by JeffreySarnoff** [comment]
fixed with merge from @dpsanders
